### DISCUSSION TITLE
Bannable or Not

### DISCUSSION THREAD

**Discussion by Aureiloon** [shalzuth/LostArkLogger#44] [General]
Yo,
I would like to support your / your Lost Ark Logger, but I have a question, is it safe? I mean for my lost ark account. Because I would love to use it for myself, improving my gameplay and if I find a bug, inform you about it, but in the back of my head I have this ... that I will lose an account on which I have already played 1000 hours

Should I wait for smile gate to tell you what it's going to do with your project?
Thanks in advance for your answer, best regards and have a nice evening / day.
-Aureiloon

**Comment by syntaxerror41** [comment]
Smilegate's community manager "Roxx" already responded in a thread talking about dps meters in general. The statement was:

"In terms of our rules, any app that modifies the game and/or interacts with it in a way that provides information that is otherwise not available to players is not approved"

People interpret this statement in various different ways. Personally, I don't think using a dps meter violates this statement. Even if it does, I think smilegate currently has more important things to worry about than people using a tool that gives ZERO advantages over others, and does not modify or inject anything to the game.

That being said, as the developers of this project themselves say, use at your own risk. Could you get banned? Maybe, personally I doubt it, but even if you did, I highly doubt it would be a permaban without warning. Just make sure you are not toxic while using it, and you don't advertise in-game that you're using a dps meter.
